developer-model-service/docs/access.md

```diff
@@ -0,0 +1,139 @@
+# Access & Usage Information
+
+## Basic Information
+
+### Model Server URL
+
+The model server can be accessed [here](https://model-service.apps.domain.com/).
+
+## Authentication
+
+In order to gain access to the model server you will need to sign in with your SSO credentials by selecting Google auth. Once you have signed in you are able to generate a token by navigating to **`Apps and API Keys`** on the top ribbon and hitting **`Create new Application`**:
+
+![Generation Example](../images/generation-example.png)
```

Comment:
Since we moved `access.md` from `/code` directory this will need to be updated as well 
```suggestion
![Generation Example](./images/generation-example.png)
```

Reply:
Updated